tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

```diff
@@ -289,16 +322,30 @@ export class LineChartComponent
     return false;
   }
 
+  private onContextLost() {
+    // Since context may be lost when the component is hidden or does not need
+    // updates, the teardown and recreation happens lazily.
+    this.isRenderingContextLost = true;
```

Comment:
For some reason, I thought you'd dispose `lineChart` here and set it to `this.lineChart = null`. Would that make the code more readable in anyway?

Reply:
Done, yes a bit more expected.